source/tomopy/misc/corr.py

```diff
@@ -267,6 +268,137 @@ def median_filter_cuda(arr, size=3, axis=0):
     return out
 
 
+def find_nonfinite_values_prj(projection, slc_idx):
```

Comment:
```suggestion
def _find_nonfinite_values_prj(projection, slc_idx):
```
Functions that are not intended for external use should have a leading underscore.